terraform-modules/aws/msk/outputs.tf

```diff
@@ -0,0 +1,15 @@
+output "cluster_name" {
+  description = "MSK Cluster name"
+  value       = module.msk.cluster_name
+}
+
+
+output "cluster_arn" {
+  description = "Amazon Resource Name (ARN) of the MSK cluster"
+  value       = module.msk.cluster_arn
+}
+
+output "zookeeper_connect_string" {
+  description = "A comma separated list of one or more hostname:port pairs to use to connect to the Apache Zookeeper cluster"
+  value       = module.msk.zookeeper_connect_string
+}
```

Comment:
I think we need some additional outputs?  How are you getting the bootstrap servers?  

Probably add a few more since we are working with this?  

https://github.com/cloudposse/terraform-aws-msk-apache-kafka-cluster#outputs
* [bootstrap_brokers](https://github.com/cloudposse/terraform-aws-msk-apache-kafka-cluster#output_bootstrap_brokers)
* [bootstrap_brokers_iam](https://github.com/cloudposse/terraform-aws-msk-apache-kafka-cluster#output_bootstrap_brokers_iam)
* [bootstrap_brokers_scram](https://github.com/cloudposse/terraform-aws-msk-apache-kafka-cluster#output_bootstrap_brokers_scram)
* [bootstrap_brokers_tls](https://github.com/cloudposse/terraform-aws-msk-apache-kafka-cluster#output_bootstrap_brokers_tls)
* [current_version](https://github.com/cloudposse/terraform-aws-msk-apache-kafka-cluster#output_current_version)
* [hostname](https://github.com/cloudposse/terraform-aws-msk-apache-kafka-cluster#output_hostname)
*

Reply:
bootstrap servers are getting by devs by aws cli
but you're right we can add more for future proposals. 
working on it

Reply:
I attended your comment
I added more outputs

```
Changes to Outputs:
  + all_brokers             = (known after apply)
  + bootstrap_brokers_iam   = "b-1.exactpaydpdev.lxr72p.c12.kafka.us-west-2.amazonaws.com:9098,b-2.exactpaydpdev.lxr72p.c12.kafka.us-west-2.amazonaws.com:9098"
  + bootstrap_brokers_scram = "b-1.exactpaydpdev.lxr72p.c12.kafka.us-west-2.amazonaws.com:9096,b-2.exactpaydpdev.lxr72p.c12.kafka.us-west-2.amazonaws.com:9096"
  + config_arn              = "arn:aws:kafka:us-west-2:***:configuration/exactpay-dp-dev/af947729-b9b0-4657-b00a-72fe25df9954-12"
  + current_version         = "K2M66E0AN2PS0S"
  + hostname                = ","
  + latest_revision         = "1"
  + security_group_id       = "sg-0c0d870db2ad39276"
  + security_group_name     = "exactpay-dp-dev"
```